resreporter/mem_linux.go

```diff
@@ -0,0 +1,70 @@
+package resreporter
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"os"
+)
+
+const meminfo = "/proc/meminfo"
+
+var Memory = memory{}
+
+type memory struct{}
+
+func (memory) Used() (used uint64, total uint64) {
+	fd, err := os.Open(meminfo)
+	if err != nil {
+		//XXX Should this use metafora's logger somehow?
+		log.Printf("Error reading free memory via "+meminfo+": %v", err)
```

Comment:
This makes me think that perhaps the interface should include an error value? Then the balancer can explicitly handle them, as opposed to this reporter essentially disabling the balancer outside of its scope?